### ISSUE TITLE
Sanguine 

### ISSUE THREAD

**Issue by beanaboston** [elucent/eidolon#48]

**Comment by elucent** [comment]
I have heard this is a mod conflict. The person who last reported it in my Discord said it was a conflict with the DungeonsGear mod that caused the issue - do you have that mod installed, and does removing it resolve the issue?

**Comment by froehlichA** [comment]
Yes, we have that mod. Removing it resolves the issue.
Is this something you'll be able to get fixed on your side, or should I create an issue in the Eidolon repo?

**Comment by elucent** [comment]
I'm not sure what the cause is, I suspect the Dungeons Gear developer might be better able to figure it out? I'd say, you should report the issue on their issue tracker, and if it turns out it's easier to fix on my end I'd be open to it.

**Comment by froehlichA** [comment]
I've created a new issue on the Dungeon Gear's Github [here](https://github.com/Thelnfamous1/Dungeons-Gear/issues/47), and I noticed every faulty Eidolon recipe uses a potion as an ingredient. Maybe something's off with that?

**Comment by elucent** [comment]
Yes, that seems likely. But I'm not sure what it'd be. I check that the potion stack is the same as in the recipe on my end, so minor changes in potion NBT could potentially break the recipe. However, I wasn't able to figure out what Dungeons Gear might be doing to cause that from a cursory look at the repository. If they make notable changes to vanilla potion NBT, that might explain this issue.

**Comment by Thelnfamous1** [comment]
I don’t do anything myself with potions except during events

**Comment by Canine-Medal** [comment]
put the potion in the necrotic focus in stead of the necklace, which goes in the brazier
